From fbc2bf3242768fd90e0929b830277c802cfb6309 Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Mon, 4 Nov 2019 11:09:05 +0800
Subject: [PATCH] dt-bindings: usb: dwc3: add dis-u3-autosuspend-quirk property

This patch adds a new property "snps,dis-u3-autosuspend-quirk" for
xHCI integrated in DWC3 IP to disable USB 3.0 root HUB autosuspend.

Change-Id: Ibaf2b8d0e0472b052d3ab46010b7477274f4bb78
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 Documentation/devicetree/bindings/usb/dwc3.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/usb/dwc3.txt b/Documentation/devicetree/bindings/usb/dwc3.txt
index 6a0d54b3bbfa..5945bf5452e7 100644
--- a/Documentation/devicetree/bindings/usb/dwc3.txt
+++ b/Documentation/devicetree/bindings/usb/dwc3.txt
@@ -59,6 +59,8 @@ Optional properties:
  - snps,tx_de_emphasis_quirk: when set core will set Tx de-emphasis value.
  - snps,tx_de_emphasis: the value driven to the PHY is controlled by the
 			LTSSM during USB3 Compliance mode.
+ - snps,dis-u3-autosuspend-quirk: when set USB core driver will disable
+			USB3 autosuspend function.
  - snps,dis_u3_susphy_quirk: when set core will disable USB3 suspend phy.
  - snps,dis_u2_susphy_quirk: when set core will disable USB2 suspend phy.
  - snps,dis-u1u2-quirk: when set core will reject transition to U1 or U2.
-- 
2.35.3

